blog.html

```diff
@@ -5,21 +5,88 @@
     	<meta name="description" content="Your description goes here">
     	<meta name="keywords" content="one, two, three">
 
-		<title>Blank Template</title>
+		<title>Relaxr Blog</title>
 
 		<!-- external CSS link -->
 		<link rel="stylesheet" href="css/reset.css">
 		<link rel="stylesheet" href="css/style.css">
+		<link href='https://fonts.googleapis.com/css?family=Open+Sans:300,300italic,600,800' rel='stylesheet' type='text/css'>
 
 	</head>
 	<body>
 		<div id="container">
 			<header>
-				<h1>Blank Template</h1>
-				<h2>Just another blank template</h2>
+				<nav class="navbar clearfix">
+					<h1 class="hlogo">Relaxr</h1>
+					<div class="navlinkgroup">
+						<ul class="navlinks">
+							<li><a href="index.html" class="headerlinks">About</a></li>
+							<li><a href="" class="headerlinks">FAQ</a></li>
+							<li><a href="" class="headerlinks">Team</a></li>
+							<li><a href="" class="headerlinks">Contact Us</a></li>
+							<li><a href="blog.html" class="headerlinks">Blog</a></li>
+						</ul>
+					</div>
+				</nav>
 			</header>
-			<footer>
-				&copy; Footer Goes Here
+			<main class="contentbackground clearfix">
```

Comment:
I would probably call the `main` element here a `div`, and then have the two floated items be `main` and `aside`. https://developer.mozilla.org/en-US/docs/Web/HTML/Element/main You generally don't want navigation links in sidebars as part of a `main` element.